### ISSUE TITLE
Placeholders in input fields are not readable when a dark background is used.

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyone#623]
**Describe the bug**

Regular input fields has a background color that is the same as the body background color, but with lower opacity.

When a dark body background color is used, the _placeholder_ text inside form input fields does not have a high enough contrast against the background.

**To Reproduce**
1. Change the background color
2. Add an input field with a placeholder. _-In my case, I was looking at a coupon field of a WooCommerce install._
But I was able to reproduce the same, by adding a search block with a placeholder in the editor.
3. See error

**Expected behavior**
The text needs to be readable.

**Screenshots**

Search block placeholder in the editor and front:
![search](https://user-images.githubusercontent.com/7422055/96680147-41bdef00-1375-11eb-8a45-0963341cf92a.png)

Example placeholder with WooCommerce installed:
![placeholder](https://user-images.githubusercontent.com/7422055/96679941-e12eb200-1374-11eb-907a-277565d1cdb5.png)

**Editor version (please complete the following information):**
- WordPress version: (5.6-beta1-49262)
- If the Gutenberg plugin is installed, which version is it? 9.2
(WooCommerce 4.6.0)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome Version 86.0.4240.75

**Comment by carolinan** [comment]
The problem with always using the white background color for the input field instead of the current _body background color with a lower opacity_,  is that the growing, white border that is used as the focus style becomes pretty much invisible. 🤔
